app/views/events/participants.html.erb

```diff
@@ -32,6 +23,13 @@
       <%= participant.email %>
     </td>
 	<td>
+	  <% if participant.older_than_18_at_start_date_of_event?(@event) %>
+	    <p class="text-muted"> <%= t '.unnecessary' %> </p>
```

Comment:
You should put these in a &lt;span> instead of a &lt;p> tag, currently it messes up the table.